specs/commandline/sign.md

```diff
@@ -0,0 +1,65 @@
+# notation sign
+## Description
+Use `notation sign` to sign artifacts.
+
+Signs a container artifact that is stored in a registry. Upon successful signing, the generated signature is pushed to the registry and the digest of the container image is returned.
+## Outline
+```
+Sign artifacts
+
+Usage:
+  notation sign <reference> [flags]
+
+Flags:
+      --cert-file string        Location of file containing signing(leaf) certificate and certificate chain. Use this flag with flag '--key-file' together.
+  -e, --expiry duration         Optional expiry that provides a “best by use” time for the artifact. The duration is specified in seconds, minutes or hours.
```

Comment:
IMO there is no real world usecase where customer would want an artifact to expire in second.
```suggestion
  -e, --expiry duration         Optional expiry that provides a “best by use” time for the artifact. The duration is specified in minutes, hours or days.
```